app/src/main/java/org/mozilla/fenix/settings/PrivateBrowsingFragment.kt

```diff
@@ -41,5 +43,17 @@ class PrivateBrowsingFragment : PreferenceFragmentCompat() {
         findPreference<SwitchPreference>(getPreferenceKey(R.string.pref_key_open_links_in_a_private_tab))?.apply {
             onPreferenceChangeListener = SharedPreferenceUpdater()
         }
+
+        findPreference<Preference>(getPreferenceKey(R.string.pref_key_allow_screenshots_in_private_mode))?.apply {
+            setOnPreferenceClickListener {
+                ScreenshotsGuardian().invoke(requireActivity() as AppCompatActivity)
```

Comment:
We'd not need this cast if ScreenshotsGuardian's methods would take `FragmentActivity` as a parameter.